spec/models/piece_spec.rb

```diff
@@ -94,4 +94,37 @@ def create_chess(x, y)
       end
     end
   end
+
+  describe '#on_board?' do
+    it 'should return true is the coords are from 1 to 8' do
+      piece = Piece.create
+      (1..8).each do |x|
```

Comment:
rubocop tends to complain, so I stopped trying to add spaces

I can look into rubocop later